tests/unit/badge/badge.renderBadge.spec.js

```diff
@@ -0,0 +1,42 @@
+import { shallow } from 'enzyme';
```

Comment:
@EduardTrutsyk I also thought like that before - you can see from the amount of projects we have in rwd - but that will change in the future specially for rwd :)

The reasons why a one file is better, is related to what we're testing.

We're testing how our component behaves when rendering and/or interacting with it (simulating click events, etc).

This is definitely a good way to test integration between apps, and test if the app's functionality is broken.

I don't see that in our approach on rwd. We test really well our functions, but that only gives us a good coverage for each unit, not a proper sign that we're breaking the intended functionality or changing it's behaviour :)